x/emissions/keeper/inference_synthesis/network_regrets.go

```diff
@@ -377,9 +380,12 @@ func GetCalcSetNetworkRegrets(args GetCalcSetNetworkRegretsArgs) error {
 		}
 	}
 
-	// Recalculate topic initial regret
-	if len(workersRegrets) > 0 {
-		updatedTopicInitialRegret, err := CalcTopicInitialRegret(
+	// Check if we have enough workers with enough experience to calculate the topic initial regret
+	// If we don't have enough workers with enough experience, use the fallback regrets
+	var updatedTopicInitialRegret alloraMath.Dec
+	var err error
+	if len(workersRegrets) >= 10 {
```

Comment:
Must we always be limited to those regrets passed in this epoch?

Wondering if it makes more sense to keep a running, recent history of regrets like we do for scores @jmdkastro ?

For reference, see how `MaxSamplesToScaleScores` is used to see how we maintain a running, recent history of numbers.

Reply:
Regrets are already an EMA, so there it isn't necessary. Scores are instantaneous, so there it makes more sense to have some running history.